scripts/quests/bastok/Father_Figure.lua

```diff
@@ -20,7 +20,9 @@ quest.sections =
         check = function(player, status, vars)
             return status == xi.questStatus.QUEST_AVAILABLE and
                 player:hasCompletedQuest(xi.questLog.BASTOK, xi.quest.id.bastok.THE_ELVAAN_GOLDSMITH) and
-                player:getFameLevel(xi.fameArea.BASTOK) >= 2
+                player:getFameLevel(xi.fameArea.BASTOK) >= 2 and
+                player:getQuestStatus(xi.questLog.SANDORIA, xi.quest.id.sandoria.DISTANT_LOYALTIES) ~= xi.questStatus.QUEST_ACCEPTED
```

Comment:
Not sure if this is really the case, or if we should be using event priority more effectively.  Not a problem for now, but a bigger discussion in the future.

Reply:
The capture wouldn't let me flag quest while I had it loaded. Uses the same NPC - spoke to it multiple times, but I agree there might be some priority for others...